src/game.py

```diff
@@ -111,22 +218,68 @@ def read_data(file_name: str) -> list[dict[str, str]]:
     return db_list
 
 
+def give_character_choice(characters: list[dict[str, str]]) -> dict[str, str]:
+    """Give the user a choice of characters to play with."""
+    # printing out the characters
+    print("==================================================")
+    print("Here are the characters you can choose to be your champion: \n")
+    for i, character in enumerate(characters):
+        print(f"{i+1}. {character['Name']}")
+
+    # getting user input for character choice
+    while True:
+        try:
+            character_choice = int(
+                input("\nPlease enter the number of your champion: ")
+            )
+            if character_choice > len(characters) or character_choice < 1:
+                print("Please enter a valid number!")
+            else:
+                print("May your champion make it to the end!\n")
+                print("==================================================")
+                break
+        except ValueError:
+            print("Please enter a valid number!")
+
+    # returning the character choice
+    return characters[character_choice - 1]
+
+
 # reading in data for characters and spells
-hp_characters = "../data/Harry_Potter_Characters.tsv"
+hp_characters = (
```

Comment:
this path will only work for your computer so it should be the path to the data in the repo (as it was previously)